pkg/cmd/validate.go

```
@@ -578,3 +592,4 @@ func Context(fileContext file.Context) (cont *base.Context, err error) {
 	// If everything goes well, return the context and a nil error.
 	return cont, nil
 }
+
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofumpt)